### DISCUSSION TITLE
Which US president became the first American to be awarded the Nobel Prize?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#662] [Galaxy Trivia] [answered]
**Bonus:** How many U.S. presidents have received this honor?

**Comment by peterbaccaro** [comment]
The first American president to be awarded the Nobel Prize was **Theodore Roosevelt**. 

A total of **five U.S. presidents** have received the Nobel Peace Prize:

Theodore Roosevelt (1906)
Woodrow Wilson (1919)
Jimmy Carter (2002)
Barack Obama (2009)
Joe Biden (2021)

**Reply by ParanoidUser** [reply]
Not all of the abovementioned were found in the list https://www.nobelprize.org/prizes/lists/all-nobel-prizes. Is there any other source to check?

**Reply by peterbaccaro** [reply, marked as answer]
I think actually only **four U.S. presidents** have received the Nobel Peace Prize:

Theodore Roosevelt (1906)
Woodrow Wilson (1919)
Jimmy Carter (2002)
Barack Obama (2009)

https://www.biography.com/political-figures/a43031160/presidents-who-won-the-nobel-peace-prize
